src/SearchResponse.swift

```diff
@@ -12,12 +12,30 @@ import Pelias
 /// Represents a response for a request executed by 'MapzenSearch'
 @objc(MZSearchResponse)
 public class SearchResponse : NSObject {
+
   let peliasResponse: PeliasResponse
 
   private lazy var internalParsedResponse: ParsedSearchResponse? =  { [unowned self] in
     guard let peliasParsedResponse = self.peliasResponse.parsedResponse else { return nil }
     return ParsedSearchResponse.init(peliasParsedResponse)
   }()
+
+  private static let validTypes: Set = ["Point", "MultiPoint", "LineString", "MultiLineString", "Polygon", "MultiPolygon", "GeometryCollection", "Feature", "FeatureCollection"]
+
+  private lazy var internalIsGeoJson: Bool = { [unowned self] in
```

Comment:
I agree with returning an error in that case...I had pushed [this change](https://github.com/pelias/pelias-ios-sdk/pull/75) which will return errors if we dont get valid geojson. When it merges we can update pelias...i will remove this variable since it shouldnt be needed, the presence of a parsed response will mean we got valid geojson